### PR TITLE
fix: miss pods/log in workspace role templates

### PR DESCRIPTION
**Pull request by huanggze**
Signed-off-by: huanggze <loganhuang@yunify.com>

image

```
$ kubectl get  globalrole  role-template-manage-workspaces  role-template-view-workspaces
NAME                              AGE
role-template-manage-workspaces   3d
role-template-view-workspaces     3d
```

### CODE REVIEW

**Comment by benjaminhuo** [comment]
updates to the previous PR?

**Comment by huanggze** [comment]
The previous PR has been merged...

File: roles/ks-core/prepare/files/ks-init/iam-accounts.yaml [hunk 1]
```diff
@@ -622,6 +622,7 @@ rules:
   - ingresses
   - filters
   - pods
+  - pods/log
```
Comment (rayzhou2017) [inline roles/ks-core/prepare/files/ks-init/iam-accounts.yaml:625]: How about events, audits?

Comment (benjaminhuo) [reply roles/ks-core/prepare/files/ks-init/iam-accounts.yaml:625]: audits and events don't have the problem

Comment (huanggze) [reply roles/ks-core/prepare/files/ks-init/iam-accounts.yaml:625]: The resource name for events is just `events`. Logs are subresources of pods. They are different.

By now, events and auditing share authorizaton management model. If one has access to events, this person can view audting as well.

**Pull request merged**